infrastructure/state.tf

```diff
@@ -1,17 +1,10 @@
+provider "azurerm" {
+  features {}
+  skip_provider_registration = true
+  alias                      = "postgres_network"
+  subscription_id            = var.aks_subscription_id
+}
+
 terraform {
   backend "azurerm" {}
-
-  required_providers {
```

Comment:
why remove all these?

Reply:
Rectified in the latest commit, Thanks for picking this up.